dev/tasks/java-jars/github.yml

```diff
@@ -140,6 +140,12 @@ jobs:
           brew uninstall protobuf
 
           brew bundle --file=arrow/java/Brewfile
+          
+          # We want to use the bundled googletest for static linking. Since
+          # both BUNDLED and brew options are enabled, it could cause a conflict
+          # when there is a version mismatch.
+          # We uninstall googletest to ensure using the bundled googletest.
+          brew uninstall googletest
```

Comment:
Could you move this part before `brew bundle --file=arrow/java/Brewfile`?
Because `googletest` is in `cpp/Brewfile` not `java/Brewfile`.

Reply:
Sure @kou I will make a follow up PR shortly.

Reply:
@kou PR is here: https://github.com/apache/arrow/pull/43462